build_library/check_root

```diff
@@ -70,13 +70,36 @@ IGNORE_MISSING = {
                                  SonameAtom("x86_64", "libebt_vlan.so"),
                                  SonameAtom("x86_64", "libebtable_broute.so"),
                                  SonameAtom("x86_64", "libebtable_filter.so"),
-                                 SonameAtom("x86_64", "libebtable_nat.so")],
+                                 SonameAtom("x86_64", "libebtable_nat.so"),
+                                 SonameAtom("ppc_64", "libebtable_nat.so"),
+                                 SonameAtom("ppc_64", "libebt_802_3.so"),
+                                 SonameAtom("ppc_64", "libebt_among.so"),
+                                 SonameAtom("ppc_64", "libebt_arp.so"),
+                                 SonameAtom("ppc_64", "libebt_arpreply.so"),
+                                 SonameAtom("ppc_64", "libebt_ip.so"),
+                                 SonameAtom("ppc_64", "libebt_ip6.so"),
+                                 SonameAtom("ppc_64", "libebt_limit.so"),
+                                 SonameAtom("ppc_64", "libebt_log.so"),
+                                 SonameAtom("ppc_64", "libebt_mark.so"),
+                                 SonameAtom("ppc_64", "libebt_mark_m.so"),
+                                 SonameAtom("ppc_64", "libebt_nat.so"),
+                                 SonameAtom("ppc_64", "libebt_nflog.so"),
+                                 SonameAtom("ppc_64", "libebt_pkttype.so"),
+                                 SonameAtom("ppc_64", "libebt_redirect.so"),
+                                 SonameAtom("ppc_64", "libebt_standard.so"),
+                                 SonameAtom("ppc_64", "libebt_stp.so"),
+                                 SonameAtom("ppc_64", "libebt_ulog.so"),
+                                 SonameAtom("ppc_64", "libebt_vlan.so"),
+                                 SonameAtom("ppc_64", "libebtable_broute.so"),
+                                 SonameAtom("ppc_64", "libebtable_filter.so")],
 }
 
-USR_LINKS = ("/bin/", "/sbin/", "/lib/", "/lib32/", "/lib64/")
+#USR_LINKS = ("/bin/", "/sbin/", "/lib/", "/lib32/", "/lib64/")
+USR_LINKS = ("/sbin/", "/lib/", "/lib32/")
 
 IGNORE_SHEBANG = (
     b"*/python2.7/cgi.py",
+    b"*/python3.4/cgi.py",
```

Comment:
Hi @marineam,
>> Use powerpc64le-cros-linux-gnu for the board CHOST, it helps ensure cross-compiling >> >> works even if using a ppc64le SDK.

Now We've switched to powerpc64le-cros-linux-gnu for the board CHOST and cross building all packages for ppc64le-usr board. 

Currently We're facing some cross building issue.
 e.g. https://bugs.gentoo.org/show_bug.cgi?id=593484

Util finished the build_package for /build/ppc64le-usr, I'll give you updated status. Thanks